### PR TITLE
new: dev: Add support for ClusterManage Command

### PR DESCRIPTION
**Pull request by elricL**
Addressed the comments in pull request #91 and opening on against unreleased branch.

### CODE REVIEW

**Comment by vrajat** [comment]
Looks good to me other than indentation. Travis complains that there is a mix of space and tabs. Surprised it didnt fail in 2.X

File: qds_sdk/cluster.py [hunk 1]
```diff
@@ -421,7 +447,7 @@ def _parse_snapshot_restore_command(cls, args, action):
         Parse command line arguments for snapshot command.
         """
         argparser = ArgumentParser(prog="cluster %s" % action)
-        
+
```
Comment (mindprince) [inline qds_sdk/cluster.py]: Get rid of these whitespace changes.

Comment (mindprince) [inline qds_sdk/cluster.py]: However, if these are trailing whitespace removals - then they are welcome.

**Comment by elricL** [comment]
Fixed all the indentation issues(Finally). Can this be merged now?

**Comment by vrajat** [comment]
Oh oh - I messed up. Didnt squash it

**Comment by elricL** [comment]
Shall I rebase everything here? so you can reset hard and merge?

**Pull request merged**